lib/pushmi_pullyu/aip.rb

```diff
@@ -1,63 +1,36 @@
-require 'pushmi_pullyu'
-require 'bagit'
-require 'archive/tar/minitar'
+require 'fileutils'
 
 module PushmiPullyu::AIP
-  # Exceptions
-  class BagInvalid < StandardError; end
-  class NoContentFilename < StandardError; end
-  class FedoraFetchError < StandardError; end
```

Comment:
Move these to where they are being used. Helps with maintainability. If we decide to remove FedoraFetcher class one day, this doesn't get orphaned and forgotten about.